### DISCUSSION TITLE
SPANISH (SPAIN) TRANSLATION

### DISCUSSION THREAD

**Discussion by agora-coffee** [lobehub/sd-webui-lobe-theme#443] [General]
Hello, I would like to collaborate on the translation into Spanish (Spain) of the interface for A1111. Can you give me instructions? My email is: cgbaena@gmail.com

Carlos Baena

**Comment by canisminor1990** [comment]
Hi @agora-coffee , you can collaborate on locals follow steps below ❤️
1. Add locals file `xx_XX.json` here: https://github.com/lobehub/sd-webui-lobe-theme/tree/main/locales
2. Then add options here: https://github.com/lobehub/sd-webui-lobe-theme/blob/main/locales/options.json
